Notebooks/Russian Twitter Corpus/.ipynb_checkpoints/rusain-corpus-tweets-analysis-checkpoint.ipynb

```diff
@@ -0,0 +1,580 @@
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>3</th>\n",
+       "      <th>size</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>хоть я и школота но поверь у нас то же самое D...</td>\n",
+       "      <td>89</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>Да все таки он немного похож на него Но мой ма...</td>\n",
+       "      <td>74</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>RT Ну ты идиотка я испугалась за тебя</td>\n",
+       "      <td>47</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>RT Кто то в углу сидит и погибает от голода а ...</td>\n",
+       "      <td>134</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>Вот что значит страшилка D Но блин посмотрев в...</td>\n",
+       "      <td>115</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>...</th>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>114906</th>\n",
+       "      <td>Спала в родительском доме на своей кровати Про...</td>\n",
+       "      <td>82</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>114907</th>\n",
+       "      <td>RT Эх Мы немного решили сократить путь сейчас ...</td>\n",
+       "      <td>129</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>114908</th>\n",
+       "      <td>Что происходит со мной когда в эфире proactive...</td>\n",
+       "      <td>122</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>114909</th>\n",
+       "      <td>Любимая я подарю тебе эту звезду Имя какой зв...</td>\n",
+       "      <td>90</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>114910</th>\n",
+       "      <td>посмотри непытайтесьпокинутьомск сегодня в Вав...</td>\n",
+       "      <td>81</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>114911 rows × 2 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                                                        3  size\n",
+       "0       хоть я и школота но поверь у нас то же самое D...    89\n",
+       "1       Да все таки он немного похож на него Но мой ма...    74\n",
```

Comment:
Now let's cast out texts to lower case and re-calculate our statistics without n-gram duplicates :)